android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java

```diff
@@ -87,6 +87,7 @@ public void run() {
                 dayRecyclerView.post(new Runnable() {
                     @Override
                     public void run() {
+                        if (dayRecyclerView != null && noSchedule!=null) {
```

Comment:
This is the only change required.
Please send a new pull request with only this file changed.